### ISSUE TITLE
about marn

### ISSUE THREAD

**Issue by Harvey-Mei** [nasib-ullah/video-captioning-models-in-Pytorch#2]
hello， 
Thanks for your excellent code!
I tried to use you code to run the MARN model, but some of the hyperparameters can not be found in the class named ConfigMARN, such as amd_bottleneck_size. Could you provide a sample code for running this model, as well as the method to extra motion features.

**Comment by nasib-ullah** [comment]
Hi, 
Thanks for pointing it out. I have updated config.py, video_captioning.ipynb (demo notebook), and model.py (for MARN) to the latest version. Hope it will help you to solve the issue. For the motion feature extraction, I have followed a  repo mentioned in the acknowledgment section. The beam decoding for MARN still needs to fix for stage-2 (with attended memory decoder) of training, although greedy search performed better than the beam. The optimal hyperparameters and pre-trained models will be published soon. Write me back if you have any more doubts.
